src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -234,6 +234,10 @@
       <Context>Project</Context>
     </PropertyPageSchema>
 
+    <PropertyPageSchema Include="$(ManagedXamlNeutralResourcesDirectory)SuggestedWorkload.xaml">
+      <Context>Project</Context>
```

Comment:
This context does not match that of the export (`ProjectSubscriptionService`).

Is this project item needed for exported rules?

Reply:
> This context does not match that of the export (`ProjectSubscriptionService`).
> 
> Is this project item needed for exported rules?

I don't think it is. I have removed it.